exchange/exchange_test.go

```diff
@@ -56,28 +56,30 @@ func TestNewExchange(t *testing.T) {
 //
 // The "known" file names right now are "banner.json" and "video.json". These files should hold params
 // which the Bidder would expect on banner or video Imps, respectively.
-func TestRaceIntegration(t *testing.T) {
```

Comment:
Could alternatively do a `t.SkipNow()` with a `//TODO` comment

Reply:
much better idea ^^.

I don't think config makes sense here... this isn't something which varies from one deployment to another. `t.SkipNow` is much cleaner though